src/app/machines/views/MachineList/MachineListTable/OwnerColumn/OwnerColumn.test.tsx

```diff
@@ -57,26 +50,11 @@ describe("OwnerColumn", () => {
     });
   });
 
-  it("renders", () => {
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-        >
-          <OwnerColumn onToggleMenu={jest.fn()} systemId="abc123" />
-        </MemoryRouter>
-      </Provider>
-    );
-
-    expect(wrapper.find("OwnerColumn")).toMatchSnapshot();
-  });
-
   it("displays owner", () => {
     state.machine.items[0].owner = "user1";
     renderWithBrowserRouter(
-      <OwnerColumn onToggleMenu={jest.fn()} showFullName systemId="abc123" />,
-      { state }
+      <OwnerColumn onToggleMenu={jest.fn()} systemId="abc123" />,
```

Comment:
This is now missing the `showFullName` prop.
```suggestion
      <OwnerColumn onToggleMenu={jest.fn()} showFullName systemId="abc123" />,
```

Reply:
That was deliberate, since the test case below is checking for the full name, whereas this one should be checking for username. Otherwise we have two back-to-back test cases that test for the same thing

Reply:
Not necessarily, previously this test was verifying whether username is displayed even if `showFullName` is set to true. I'd assume this was done for a reason (although I could be wrong).

In any case, would be good to update the test description to `displays owner's username`.